test/unistd/access.c

```diff
@@ -65,14 +66,73 @@ int main() {
   printf("F_OK(%s): %d\n", "renamedfile", faccessat(AT_FDCWD, "renamedfile", F_OK, 0));
   printf("errno: %d\n", errno);
 
+  struct stat fileStats;
+  stat("fchmodtest", &fileStats);
+#if !defined(NODEFS) && !defined(NODERAWFS)
```

Comment:
Please add a comment explaining why node behaves differently here. I actually don't know why, myself. It is a bug in the node code, or the non-node code, or expected for some reason?